core/identity-management/iin-agent/src/fabric-ledger/config.json.template

```diff
@@ -11,5 +11,8 @@
     },
     "mspId":"Org1MSP",
     "ordererMspIds": ["OrdererMSP"],
-    "caUrl":""
+    "ccpPath": "path-to-connection-profile",
```

Comment:
Put angular brackets around the value. That's the general convention we use in other places whenever the given value is meant to be a variable rather than a constant.